lisa/tools/kdump.py

```diff
@@ -98,15 +98,30 @@ def _install_from_src(self) -> None:
         kexec_tar = wget.get(self._kexec_repo, str(tool_path))
         tar = self.node.tools[Tar]
         tar.extract(kexec_tar, str(tool_path))
-        kexec_source = tar.get_root_folder(kexec_tar)
-        code_path = tool_path.joinpath(kexec_source)
+        find_tool = self.node.tools[Find]
+        kexec_source_folder = find_tool.find_files(
+            tool_path, name_pattern="kexec-tools*", type="d"
+        )
+        code_path = tool_path.joinpath(kexec_source_folder[0])
```

Comment:
the tar ball has a fixed name, we can use it directly?

Reply:
We don't need to care about the name because it may vary. Using 'd' type will get the folder after unzipping more accurately.